server/actions/serveFile.js

```diff
@@ -128,7 +135,6 @@ function serveStaticFile(req, res) {
 
   // Cache files for 1 year.
   res.set({
-    "Content-Type": contentType,
```

Comment:
If you're not using the `contentType` at all (as it seems), you can delete the code above that gets and adjusts the variable (which is now unused).